infra/docker/python_base/Dockerfile

```diff
@@ -34,3 +36,4 @@ RUN python3 setup.py install
 WORKDIR /opt
 ENV PYTHONPATH /opt
 RUN rm -rf filter_lib && rm -rf tenant_dependency
+RUN pip uninstall certifi -y
```

Comment:
Is this relevant to the issue you're working on?

Reply:
@serereg

Reply:
Fixed